relay-server/src/actors/processor.rs

```diff
@@ -1360,6 +1383,7 @@ impl EnvelopeProcessorService {
             extracted_metrics: Default::default(),
             project_state,
             sampling_project_state,
+            _global_config: self.global_config.clone(),
```

Comment:
nit (personal preference): does the compiler scream if we use `Arc::clone` instead of `.clone`? I think it's easier to understand that way. Your call on what your preference is (and what the compiler allows hehe)

Reply:
i didn't think about doing it that way, i agree actually, since i prefer `.clone()` to signal something potentially expensive, thanks for the tip! 😄